<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAttributeTableDialog</class>
 <widget class="QDialog" name="QgsAttributeTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attribute Table</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QgsAttributeTableView" name="mView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QCheckBox" name="cbxShowSelectedOnly">
       <property name="text">
        <string>Show selected only</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbxSearchSelectedOnly">
       <property name="text">
        <string>Search selected only</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbxCaseSensitive">
       <property name="text">
        <string>Case sensitive</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mAdvancedSearchButton">
       <property name="toolTip">
        <string>Opens the search query builder</string>
       </property>
       <property name="text">
        <string>Advanced search</string>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mHelpButton">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="mRemoveSelectionButton">
       <property name="toolTip">
        <string>Unselect all</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../images/themes/default/mActionUnselectAttributes.png</normaloff>../../images/themes/default/mActionUnselectAttributes.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mSelectedToTopButton">
       <property name="toolTip">
        <string>Move selection to top</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../images/themes/default/mActionSelectedToTop.png</normaloff>../../images/themes/default/mActionSelectedToTop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+T</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mInvertSelectionButton">
       <property name="toolTip">
        <string>Invert selection</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../images/themes/default/mActionInvertSelection.png</normaloff>../../images/themes/default/mActionInvertSelection.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mCopySelectedRowsButton">
       <property name="toolTip">
        <string>Copy selected rows to clipboard (Ctrl+C)</string>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../images/themes/default/mActionEditCopy.png</normaloff>../../images/themes/default/mActionEditCopy.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mZoomMapToSelectedRowsButton">
       <property name="toolTip">
        <string>Zoom map to the selected rows (Ctrl-J)</string>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../images/themes/default/mActionZoomToSelected.png</normaloff>../../images/themes/default/mActionZoomToSelected.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+J</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mToggleEditingButton">
       <property name="toolTip">
        <string>Toggle editing mode</string>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../images/themes/default/mActionToggleEditing.png</normaloff>../../images/themes/default/mActionToggleEditing.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mDeleteSelectedButton">
       <property name="toolTip">
        <string>Delete selected features</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../images/themes/default/mActionDeleteSelected.png</normaloff>../../images/themes/default/mActionDeleteSelected.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mAddAttribute">
       <property name="toolTip">
        <string>New column</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../images/themes/default/mActionNewAttribute.png</normaloff>../../images/themes/default/mActionNewAttribute.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mRemoveAttribute">
       <property name="toolTip">
        <string>Delete column</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../images/themes/default/mActionDeleteAttribute.png</normaloff>../../images/themes/default/mActionDeleteAttribute.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mOpenFieldCalculator">
       <property name="toolTip">
        <string>Open field calculator</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../images/themes/default/mActionCalculateField.png</normaloff>../../images/themes/default/mActionCalculateField.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="searchFor">
       <property name="text">
        <string>Look for</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="query"/>
     </item>
     <item>
      <widget class="QLabel" name="searchIn">
       <property name="text">
        <string>in</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="columnBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="toolTip">
        <string>Looks for the given value in the given attribute column</string>
       </property>
       <property name="text">
        <string>&amp;Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsAttributeTableView</class>
   <extends>QTableView</extends>
   <header>qgsattributetableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mView</tabstop>
  <tabstop>columnBox</tabstop>
  <tabstop>query</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>cbxShowSelectedOnly</tabstop>
  <tabstop>cbxSearchSelectedOnly</tabstop>
  <tabstop>mAdvancedSearchButton</tabstop>
  <tabstop>mHelpButton</tabstop>
  <tabstop>mRemoveSelectionButton</tabstop>
  <tabstop>mSelectedToTopButton</tabstop>
  <tabstop>mInvertSelectionButton</tabstop>
  <tabstop>mCopySelectedRowsButton</tabstop>
  <tabstop>mZoomMapToSelectedRowsButton</tabstop>
  <tabstop>mToggleEditingButton</tabstop>
  <tabstop>mDeleteSelectedButton</tabstop>
  <tabstop>mAddAttribute</tabstop>
  <tabstop>mRemoveAttribute</tabstop>
  <tabstop>mOpenFieldCalculator</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
